torch/serialization.py

```diff
@@ -244,7 +244,7 @@ def _cpu_deserialize(obj, location):
 
 
 def validate_cuda_device(location):
-    device = torch.cuda._utils._get_device_index(location, True)
+    device = torch._utils._get_device_index(location, True)
```

Comment:
Then, the func name should be changed, as well?
Or, we should add another XPU version?

Reply:
we need to add an XPU version. In my original plan, I intend to incorporate storage-related content into the `Allocator`'s PR. Do we need to add a `validate_xpu_device` in this PR first, or integrate with `Allocator`'s PR.